datastore/elastic_search/query.py

```diff
@@ -367,7 +416,8 @@ def _get_dynamic_fuzziness_threshold(fuzzy_setting):
                      otherwise the input is returned as it is
     """
     if isinstance(fuzzy_setting, string_types):
-        if ELASTICSEARCH_VERSION_MAJOR > 6 or (ELASTICSEARCH_VERSION_MAJOR == 6 and ELASTICSEARCH_VERSION_MINOR >= 2):
+        if constants.ELASTICSEARCH_VERSION_MAJOR > 6 or \
+                (constants.ELASTICSEARCH_VERSION_MAJOR == 6 and constants.ELASTICSEARCH_VERSION_MINOR >= 2):
```

Comment:
I know this is out of scope, but if possible can you change this condition to
`if (constants.ELASTICSEARCH_VERSION_MAJOR, constants.ELASTICSEARCH_VERSION_MINOR,) >= (6, 2,):`